Source/JavaScriptCore/jit/GCAwareJITStubRoutine.h

```diff
@@ -57,31 +57,22 @@ class OptimizingCallLinkInfo;
 // list which does not get reclaimed all at once).
 class GCAwareJITStubRoutine : public JITStubRoutine {
 public:
-    GCAwareJITStubRoutine(const MacroAssemblerCodeRef<JITStubRoutinePtrTag>&);
-    ~GCAwareJITStubRoutine() override;
+    friend class JITStubRoutine;
```

Comment:
`friend` is required even though it's the parent?

Reply:
Yes, because JITStubRoutine is calling some of private functions (Impl things) in GCAwareJITStubRoutine